UR3MoveitJoints joints_input
geometry_msgs/Pose pick_pose
geometry_msgs/Pose place_pose
---
moveit_msgs/RobotTrajectory[] trajectories